src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private TaskManager manager = new TaskManager();
```

Comment:
I think that it is a great effort to create a TaskManager class to better implement the SLAP principles.